doc/source/ray-core/ray-dashboard.rst

```diff
@@ -6,74 +6,297 @@ Ray provides a web-based dashboard for monitoring and debugging Ray applications
 The dashboard provides a visual representation of the system state, allowing users to track the performance 
 of their applications and troubleshoot issues.
 
+SANG-TODO Replace the image with GIF.
+
 .. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/Dashboard-overview.png
     :align: center
 
+Here are common workflows when using the Ray dashboard.
+
+- :ref:`See the metrics graphs <dash-workflow-critial-system-metrics>` (requires Prometheus and Grafana to be deployed).
+- :ref:`See the progress of your job <dash-workflow-job-progress>`.
+- :ref:`Find the application logs or error messages of failed tasks or actors <dash-workflow-logs>`.
+- :ref:`Profile, trace dump, and visualize the timeline of the Ray jobs, tasks, or actors <dashboard-profiling>`.
+- :ref:`Analyze the CPU and memory usage of the cluster, tasks and actors <dash-workflow-cpu-memory-analysis>`.
+- :ref:`See individual state of task, actor, placement group <dash-workflow-state-apis>`, and :ref:`nodes (machines) <dash-node-view>` which is equivalent to :ref:`Ray state APIs <state-api-overview-ref>`.
+- :ref:`View the hardware utilization (CPU, GPU, memory) <dash-workflow-resource-utilization>` so that you can compare it to Ray's logical resource usages.
+
 Getting Started
 ---------------
 
-To use the dashboard, first install Ray with the proper dependencies:
+To use the dashboard, you should use the `ray[default]` installation:
 
 .. code-block:: bash
 
   pip install -U "ray[default]"
 
-You can access the dashboard through a URL printed when Ray is initialized (the default URL is **http://localhost:8265**).
+You can access the dashboard through a URL printed when Ray is initialized (the default URL is **http://localhost:8265**) or via the context object returned from `ray.init`.
 
-If you prefer to explicitly set the port on which the dashboard will run, you can pass
-the ``--dashboard-port`` argument with ``ray start`` in the command line, or you can pass the
-keyword argument ``dashboard_port`` in your call to ``ray.init()``.
+.. code-block:: python
+
+    context = ray.init()
+    print(context.dashboard_url)
 
 .. code-block:: text
 
   INFO worker.py:1487 -- Connected to Ray cluster. View the dashboard at 127.0.0.1:8265.
 
-The dashboard is also available :ref:`when using the cluster launcher <monitor-cluster>`.
+Ray cluster comes with the dashboard. See :ref:`Cluster Monitoring <monitor-cluster>` for more details.
```

Comment:
Should "cluster monitoring" link to this page instead? .../cluster/running-applications/monitoring-and-observability.html#monitoring-the-cluster-via-the-dashboard

Currently, it points to a page under vm clusters